phpdotnet/phd/Package/PHP/Web.php

```diff
@@ -211,6 +223,40 @@ protected function writeJsonIndex() {
         v("Index written", VERBOSE_FORMAT_RENDERING);
     }
 
+    protected static function generateDeprecatedInfo($filename) {
+        static $info;
```

Comment:
Ahhh. I see how versionInfo() does it now. That seems like we could have simplified this and safed tons of method calls.
I suppose at this time its not worth the troubles to fix it.


I still prefer to move this method to the same place, and I'm fine with loading the file rendering the pure html output too even though its not used.